src/components/Header/Polling.tsx

```diff
@@ -126,8 +126,43 @@ export default function Polling() {
   const eth = chainId ? WETH9[chainId] : undefined
   const [, pool] = usePool(kromToken, eth, FeeAmount.MEDIUM)
 
+  // Mainnet
   const [feeValue0, feeValue1] = useV3PositionFees(pool ?? undefined, BigNumber.from('154097'), false)
 
+  // Arbitrum
+  const [feeValue0ArbPosition1, feeValue1ArbPosition1] = useV3PositionFees(
```

Comment:
create 1 method or hook in which all of this logic will be handled. 
Try do reduce repetitive code, by creating a map with per chain id, for all position ids(see example below).Afterwards, make one call with current chain in place;

OWNER_LP_PER_CHAIN : Map = { 
10: '1234',
137: '123',
1: '12',
... 
}

Afterwards make 1 call in retrospect to current chain.